Equilobe.DailyReport.Models/ReportPolicy/User.cs

```diff
@@ -10,6 +10,7 @@ namespace Equilobe.DailyReport.Models.ReportPolicy
     public class User
     {
         public string JiraUserKey { get; set; }
+        public string JiraDisplayName { get; set; }
```

Comment:
atunci trebuie redenumite clasele:
get user returneaza o lista de useri desi conform restul claselor din care face parte ar trebui sa returneze o lista de jiraUser [getUser nu ar trebui sa returneze detalii despre useri ? cheia e folosita in scop de bind, name e folosit in scop de info pentru project manager... nu cred ca daca ii oferi un key, va sti ce e ala]